src/test/java/nextstep/utils/FakeGithubController.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.utils;
+
+import nextstep.member.application.dto.GithubAccessTokenRequest;
+import nextstep.member.application.dto.GithubAccessTokenResponse;
+import nextstep.member.application.dto.GithubProfileResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class FakeGithubController {
+
+    @PostMapping("/github/login/oauth/access_token")
+    public ResponseEntity<GithubAccessTokenResponse> accessToken(
+            @RequestBody GithubAccessTokenRequest tokenRequest) {
+        String accessToken = GithubResponse.getAccessTokenByCode(tokenRequest.getCode());
+        GithubAccessTokenResponse response = new GithubAccessTokenResponse(accessToken);
+        return ResponseEntity.ok(response);
+    }
+
+    @GetMapping("/github/user")
+    public ResponseEntity<GithubProfileResponse> user(
+            @RequestHeader("Authorization") String authorization) {
```

Comment:
이미 정의된 상수를 활용해보면 어떨까요?
```suggestion
            @RequestHeader(HttpHeaders.AUTHORIZATION) String authorization) {
```